source/_integrations/modbus.markdown

```diff
@@ -617,6 +617,39 @@ climates:
               description: "Value corresponding to HVAC Heat/Cool mode."
               required: false
               type: [integer, list]
+        hvac_mode_target_temp_reg_values:
+          description: "Mapping between HVAC modes and the target temperature register belonging to the mode. Note: only the relevant keys may be added. In case of some is missing, the target_temp_register is used as default.  "
+          required: false
+          type: map
+          keys:
+            state_off_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Off mode."
+              required: false
+              type: integer
+            state_heat_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Heat mode."
+              required: false
+              type: integer
+            state_cool_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Cool mode."
+              required: false
+              type: integer
+            state_auto_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Auto mode."
+              required: false
+              type: integer
+            state_dry_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Dry mode."
+              required: false
+              type: integer
+            state_fan_only_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Fan only mode."
+              required: false
+              type: integer
+            state_heat_cool_target_temp_register:
+              description: "Target Temperature Register value corresponding to HVAC Heat/Cool mode."
+              required: false
```

Comment:
This does not seem correct, the map is called "*values", the description of each entry talk about "Register value", but the entries are called "*register", so is it register value or register.

Reply:
I understand your point of view. They are registers. If you agree, I change the doc and, of course I change the belonging code renaming the map as hvac_mode_target_temp_registers.

Reply:
Updated with reference to the new PR